_posts/2024-01-25-WASI-Preview2.md

```diff
@@ -0,0 +1,206 @@
+---
+title: WASI Preview 2 Launched
```

Comment:
```suggestion
title: WASI 0.2 Launched
```

Reply:
I'm OK with "WASI 0.2", especially as we consistently used that terminology during last week's Plumber's Summit.  I do want to make sure we let people know early on in the post that what we are calling "WASI 0.2" here used to be widely referred to "WASI Preview 2" and is still going to be encountered using that previous nomenclature. (E.g. the repo is WASI/preview.). 

So I'm fine with WASI 0.2 in the title but think we should make that point as early in the document as possible.

Also, Eric Gregory has just staged a PR with his write-ups from both days of last week's Plumber's Summit and he uses the "Preview 2" and "Preview 3" nomenclature there.  Should we ask him to change that??

Reply:
> So I'm fine with WASI 0.2 in the title but think we should make that point as early in the document as possible.

Yes, on line 13 here I have added a mention that "WASI 0.2" is also known as "WASI Preview 2".

> Also, Eric Gregory has just staged a PR with his write-ups from both days of last week's Plumber's Summit and he uses the "Preview 2" and "Preview 3" nomenclature there. Should we ask him to change that??

Yeah. It's not absolutely necessary, but it would be good if he could change that.

> Lastly, you aren't modifying your blog post to be "0.2" instead of "Preview 2" are you @sunfishcode ??

I have now modified the post to be "0.2".

Reply:
Cool. I relayed word to @ericgregory so he can look at changing things in his posts.

Also, will the URL to your post still be https://blog.sunfishcode.online/wasi-preview2/?

Reply:
I'll keep that URL live, but I'll add a new URL based on WASI 0.2 as soon as I can figure out how to do that in my blog software here.

Reply:
The new URL is now live: https://blog.sunfishcode.online/wasi-0-2/